packages/react/src/alert-dialog/AlertDialog.tsx

```diff
@@ -0,0 +1,87 @@
+import * as RadixAlertDialog from "@radix-ui/react-alert-dialog";
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import * as styles from "./AlertDialog.css";
+type AlertDialogProps = ExtendProps<
```

Comment:
let's add a blank line here after the import statements
```suggestion
import * as styles from "./AlertDialog.css";

type AlertDialogProps = ExtendProps<
```